src/tabs/RP_Getting_Started.tab

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomTab xmlns="http://soap.sforce.com/2006/04/metadata">
```

Comment:
please change the content of our current getting started visualforce page to be just an empty shell that includes your new visualforce page.  this way customers don't get a new tab, and they don't have to do anything to get the new getting started page.